### PR TITLE
Upgrade SpotBugs, and JaCoCo v2

### PR DESCRIPTION
**Pull request by muga**
follow-up of https://github.com/treasure-data/digdag/pull/789

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -118,14 +119,15 @@ subprojects {
         options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
     }
 
-    findbugs {
+    spotbugs {
+        toolVersion = '3.1.3'
```
Comment (wreulicke) [inline build.gradle]: latest spotbugs version is 3.1.5

Comment (muga) [reply build.gradle]: thank you, let me upgrade it.

**Comment by muga** [comment]
git rebased

**Comment by muga** [comment]
Merged. thank you for your help @wreulicke

**Pull request merged**